lib/dradis/plugins/pentera/importer.rb

```diff
@@ -0,0 +1,80 @@
+module Dradis::Plugins::Pentera
+  class Importer < Dradis::Plugins::Upload::Importer
+    def self.templates
+      { evidence: 'evidence', issue: 'issue' }
+    end
+
+    # The framework will call this function if the user selects this plugin from
+    # the dropdown list and uploads a file.
+    # @returns true if the operation was successful, false otherwise
+    def import(params = {})
+      file_content = File.read(params[:file])
+
+      # Parse the uploaded file into a Ruby Hash
+      logger.info { "Parsing Pentera output from #{ params[:file] }..." }
+      @data = MultiJson.decode(file_content)
```

Comment:
is this the secure version?

Reply:
I'm using `MultiJson` for consistency with dradis-brakeman although `JSON` would also work here